GitUI/GitUICommands.cs

```diff
@@ -202,7 +202,12 @@ public bool StashPop(IWin32Window? owner)
             bool Action()
             {
                 bool success = FormProcess.ShowDialog(owner, process: null, arguments: "stash pop", Module.WorkingDir, input: null, useDialogSettings: true);
-                return success && MergeConflictHandler.HandleMergeConflicts(this, owner, false, false);
+                if (success)
+                {
+                    MergeConflictHandler.HandleMergeConflicts(this, owner, false, false);
```

Comment:
So what happens if conflicts aren't resolved?

Reply:
The merge conflict banner is shown similar to if a patch cannot be applied, rebase conflicts etc.
The grid needs to be updated regardless

The old handling (always run HandleMergeConflicts(), return true to always refresh the grid) was safer, I believe this is OK though.

Reply:
I must have missed these two when approving #8303. Since we cannot know how much git had done before the error, we should always request a refresh, i.e. always `return true`, please.